pkg/cli/admin/release/extract_tools.go

```diff
@@ -908,3 +927,90 @@ func copyAndReplace(errorOutput io.Writer, w io.Writer, r io.Reader, bufferSize
 		offset = end - writeTo
 	}
 }
+
+func findClusterIncludeConfig(ctx context.Context, installConfigPath string, restConfig *rest.Config) (requiredFeatureSet *string, profile *string, platform *string, capabilities *configv1.ClusterVersionCapabilitiesStatus, overrides []configv1.ComponentOverride, err error) {
+	clientVersion, reportedVersion, err := version.ExtractVersion()
+	if err != nil {
+		return nil, nil, nil, nil, nil, err
+	}
+	if reportedVersion == "" {
+		reportedVersion = clientVersion.String()
+	}
```

Comment:
These seem to be used only in the `installConfigPath` branch

Reply:
I'll use them there in a bit, but they're two functions now, so linters will catch any unused variables ;)